pkg/kubelet/status/manager.go

```diff
@@ -78,6 +78,8 @@ type Manager interface {
 	// SetPodStatus caches updates the cached status for the given pod, and triggers a status update.
 	SetPodStatus(pod *api.Pod, status api.PodStatus)
 
+	DeletePodStatus(uid types.UID)
```

Comment:
I am not sure this belongs to the Runtime interface or not. I'd say the internal PodStatus can be part of the Runtime, but the api.PodStatus might contain other information.